### DISCUSSION TITLE
High upstream response time to router

### DISCUSSION THREAD

**Discussion by h4rp00n33r** [AdguardTeam/AdGuardHome#6776] [Q&A]
Hey folks,

I have two questions about the upstream response time:

1. Why is the router being addressed here (192.168.188.1:53) and not only the registered DNS server?
2. What could be the reason why the response time to the addressed router is so high?

![image](https://github.com/AdguardTeam/AdGuardHome/assets/43474550/efd13c7e-a5b2-404e-b8f4-032207837594)

Thanks for your feedback!

**Comment by whyisthisbroken** [comment]
Could be the rDNS requests to ask the Network Device Host Names.

**Comment by h4rp00n33r** [comment]
Okay, disregard everything after "Hey folks..." X-D
I restarted the router two or three times and all 416 entries are timeouts...

@whyisthisbroken Thanks for your feedback anyway,
